datanode/centos/README.md

```diff
@@ -0,0 +1,131 @@
+# InterUSS Platform evaluation on CentOS
+These instructions describe the installation and usage of an InterUSS Platform data node on CentOS without the use of Docker (the preferred method).
+
+## Installation
+*These steps must only be followed once on a system.*
+
+1. Install OpenJDK 8:
+
+   ```yum -y install java-1.8.0-openjdk java-1.8.0-openjdk-devel```
+
+1. Do you have Python 2.7 installed?  Check with python --version.  If not, run commands below (many must be as administrative user).
+
+    1. *Building from source (most reliable, least likely to affect other parts of the system):*
+
+       ```yum -y update```  
+       ```yum groupinstall -y 'development tools'```  
+       ```yum install -y zlib-dev openssl-devel sqlite-devel bzip2-devel wget```  
+       ```wget http://www.python.org/ftp/python/2.7.15/Python-2.7.15.tar.xz```  
+       ```xz -d Python-2.7.15.tar.xz```  
+       ```tar -xvf Python-2.7.15.tar```  
+       ```cd Python-2.7.15```  
+       ```./configure```  
+       ```make && make altinstall```  
+       ```/usr/local/bin/python2.7 -m ensurepip --default-pip```  
+       ```/usr/local/bin/python2.7 -m pip install --upgrade pip setuptools wheel```  
+
+    1. If you did not follow the instructions in section 2i above, ensure you have pip and wget:
+
+       ```yum install python-pip```  
+       ```yum install wget```
+
+1. Create InterUSS Platform Python environment
+    1. Change into your preferred home folder (probably ```cd ~```)
+    1. Create a virtual environment:
+
+       ```virtualenv interuss_platform```
+
+        1. If you don’t have virtualenv (the above command resulted in an error):
+            1. If you installed Python as above:
+
+               ```/usr/local/bin/pip2.7 install virtualenv```
+
+            1. If you already had Python 2 and didn’t follow the steps under part 2:
+
+               ```pip install virtualenv```
+
+                * Note that the above should probably not be run as an administrative user.  If that command doesn’t work, try ```pip install virtualenv --user```
+1. Activate the interuss_platform Python environment:
+
+   ```cd interuss_platform```  
+   ```source bin/activate```
+
+1. Install Python packages:
+
+   ```pip install --no-cache-dir python-dateutil kazoo flask PyJWT djangorestframework cryptography```
+
+1. Copy InterUSS Platform files
+    1. Retrieve source files
+
+       ```yum install git```  
+       ```git clone https://github.com/wing-aviation/InterUSS-Platform```  
+       ```cd InterUSS-Platform/datanode/src```  
```

Comment:
Let's make this a symbolic link or some other way to not copy files around to different directories (perhaps change the install script to support full paths):
`ln -sf InterUSS-Platform/datanode/src/storage_api.py ./storage_api.py`
`ln -sf InterUSS-Platform/datanode/src/storage_interface.py ./storage_interface.py`